docs/proposals/README.md

```diff
@@ -0,0 +1,19 @@
+# OpenShift-Ansible Proposal Process
+
+The following process should be followed when a proposal is needed:
+
+1. Create a pull request with the initial proposal using the proposal template
+2. Notify the development team of the proposal and request feedback
+3. Review the proposal on the OpenShift-Ansible Architecture Meeting
+4. Update the proposal as needed and ask for feedback
+5. Accepted/Closed Phase
+  * If 75% or more of the active development team give the proposal a :+1: it is Approved
+  * If 50% or more of the active development team disagrees with the proposal it is Closed
+  * If the person proposing the proposal no longer wishes to continue they can request it to be Closed
+  * If none of the above is met the cycle can continue to Step 4.
+6. The current development lead(s) pull in the proposal to the docs folder and update the status
+7. The current development lead(s) update the issue with the result
+
+
+Once a proposal is approved, a Trello card should be created and scheduled for
+implementation.  Stale proposals will be closed after $some_time.
```

Comment:
@ashcrow I was attempting to include a time limit here but didn't know what to put for `$some_time`.  How long after no updates is too long before it is stale?  One month? How long should we give for feedback?  Two-weeks?

Reply:
I'd say if no comments or updates are made in a 2 week period the proposal should closed.

Reply:
This is probably a bit aggressive given how overloaded most of us are but we can adjust in the future.